frontend/src/lib/components/SideBar/SideBarFooter.svelte

```diff
@@ -17,6 +17,7 @@
 		german: m.german(),
 		dutch: m.dutch(),
 		italian: m.italian()
```

Comment:
the missing comma may be the cause of failed tests

Reply:
Ok great